### PR TITLE
Add dev runner for admin container

### PR DESCRIPTION
**Pull request by ghostwheel42**
## What type of PR?

development feature

## What does this PR do?

This adds a shell script (run_dev.sh) to run a live development environment in a container.

### CODE REVIEW

File: core/admin/run_dev.sh [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+### CONFIG
+
+DEV_NAME="${DEV_NAME:-mailu-dev}"
+DEV_DB="${DEV_DB:-}"
+DEV_PROFILER="${DEV_PROFILER:-false}"
+DEV_LISTEN="${DEV_LISTEN:-127.0.0.1:8080}"
+[[ "${DEV_LISTEN}" == *:* ]] || DEV_LISTEN="127.0.0.1:${DEV_LISTEN}"
+DEV_ADMIN="${DEV_ADMIN:-admin@example.com}"
+DEV_PASSWORD="${DEV_PASSWORD:-admin}"
```
Comment (nextgens) [inline core/admin/run_dev.sh:13]: I would stick with "letmein" as per the test server

**Review by nextgens — COMMENTED** [review]
works for me!

Great work; This is way more convenient.

File: core/admin/run_dev.sh (hunk 1 above)
Comment (ghostwheel42) [reply core/admin/run_dev.sh]: okay. done

**Review by nextgens — APPROVED** [review]
LGTM

**Comment by ghostwheel42** [comment]
bors try

**Pull request merged**